### PR TITLE
feat: create a naming convention to styled components

### PR DESCRIPTION
**Pull request by rafaelgslima**
Pascal Case as a naming convention to styled components

### CODE REVIEW

**Review by tassioFront — APPROVED** [review]

**Review by schirrel — APPROVED** [review]

**Review by felipefialho — APPROVED** [review]
🙏

File: README.md [hunk 1]
```diff
@@ -848,6 +848,25 @@ const Screen = () => (
 )
 ```
 
+### 7.5 Styled Component Naming Convention
```
Comment (felipefialho) [inline README.md:851]: Maybe we also could add a text to always use `import * as S from` to import SC? 😛

Comment (mauriciomutte) [inline README.md:851]: Please, add this topic in section table of content

**Review by mauriciomutte — APPROVED** [review]
🚀

**Review by rogigs — APPROVED** [review]

**Review by 3rdepaula — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rafaelgslima) [reply README.md:853]: @mauriciomutte, I forgot that, sorry.
Done: 6b8fbf21fd66fdc8ea6272a8466b7f2d36016c6b

Comment (rafaelgslima) [reply README.md:853]: @felipefialho perfect, I've included it: f710432d6a7ff2fc3c61a05a5d1960e1791278c4

File: README.md [hunk 2]
```diff
@@ -848,6 +850,54 @@ const Screen = () => (
...
+
+```js
+export const customText = styled.p`
+  color: 'red'
+`
+```
+
+### 7.6 Using Styled Component in React Components
+
+Import Styled Component as `S`
```
Comment (felipefialho) [inline README.md:875]: PSC @matheus-rosa-jsm

**Review by felipefialho — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (matheus-rosa-jsm) [reply README.md:875]: Cool!

**Review by matheus-rosa-jsm — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -848,6 +850,54 @@ const Screen = () => (
...
+**❌ Bad:**
+
+```tsx
+import * as Style from './styles'
+
+const MyComponent = () => (
+  <Style.CustomText>
+    text example
+  </Style.CustomText>
+)
```
Comment (mauriciomutte) [inline README.md:898]: Maybe we could add a more conventional case. What do you think?
```suggestion
import { CustomText } from './styles'

const MyComponent = () => (
  <CustomText>
    text example
  </CustomText>
)
```

Comment (rafaelgslima) [reply README.md:898]: Perfect.
Done: 495454127df8b725aa027146b5d623774cb6245c

**Pull request merged**